barter-data/src/subscription/book.rs

```diff
@@ -108,8 +109,28 @@ impl std::fmt::Display for OrderBooksL3 {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct OrderBookUpdate {
```

Comment:
DataBento provides an L3 book?

Reply:
Yep!

Reply:
L3, L2 and L1... think i only have access to L1 at the moment